corporate/lib/analytics.py

```diff
@@ -60,25 +57,59 @@ def estimate_annual_recurring_revenue_by_realm() -> Dict[str, int]:  # nocoverag
     return annual_revenue
 
 
+def estimate_remote_entity_annual_revenue(
+    plan: CustomerPlan, ledger_entry: Optional[LicenseLedger]
+) -> int:
+    if plan.tier in (
+        CustomerPlan.TIER_SELF_HOSTED_BASE,
+        CustomerPlan.TIER_SELF_HOSTED_LEGACY,
+        CustomerPlan.TIER_SELF_HOSTED_COMMUNITY,
+    ):
+        # None of these plan tiers generate revenue
+        renewal_cents = 0
+    elif plan.fixed_price is not None:
+        renewal_cents = plan.fixed_price
```

Comment:
Do fixed-price plans require math involving monthly vs annual here? I am not sure.

Reply:
No, we check the billing schedule (annual or monthly) after getting this number.